Revive Acrobat Adobe Reader recipes

n8felton opened the pull request:
Deprecates Acrobat Adobe Reader DC recipes and revives Acrobat Adobe Reader recipes.

Resolves #439

mpanighetti commented on AdobeReader/AdobeReader.munki.recipe:
```diff
@@ -1,143 +1,144 @@
-- An installs array is also generated for the above .app.
-- The Reader website states that 10.6.4 is a minimum requirement for 11.
+- Reader ships as an installer pkg that deploys a .app
+- Do to name changes, we add multiple variations on .app names to blocking_applications.
```
```suggestion
- Due to name changes, we add multiple variations on .app names to blocking_applications.
```

mpanighetti commented on AdobeReader/AdobeReader.munki.recipe:
```diff
@@ -1,143 +1,144 @@
+		</dict>
+	</dict>
+	<key>MinimumVersion</key>
+	<string>0.2.0</string>
```
```suggestion
	<string>1.4</string>
```

n8felton replied:
I'm not against updating the MinimumVersion on these, but I'm curious if it's _needed_, and if it is, how you came to version 1.4.

mpanighetti replied:
Because it's the minimum version for the parent recipe, so theoretically all child recipes would at least require the same minimum version. It's possible that the download recipe has an incorrect assumption on minimum version in which case it could be updated, at which point the others could be adjusted to use it as the baseline.

Probably not an essential/blocking issue, just something I noticed while looking at the changes.

n8felton replied:
Interesting observation, thanks for pointing it out. I'm good with bumping these all to 1.4.

mpanighetti commented on AdobeReader/AdobeReader.pkg.recipe:
```diff
@@ -1,47 +1,87 @@
+		<string>Mac OS 12.0</string>
+	</dict>
+	<key>MinimumVersion</key>
+	<string>0.2.0</string>
```
```suggestion
	<string>1.4</string>
```

n8felton commented:
```
% autopkg run -vvvv AdobeReader/AdobeReader.munki.recipe      
Processing AdobeReader/AdobeReader.munki.recipe...
WARNING: AdobeReader/AdobeReader.munki.recipe is missing trust info and FAIL_RECIPES_WITHOUT_TRUST_INFO is not set. Proceeding...
{'AUTOPKG_VERSION': '2.7',
 'MUNKI_REPO': '/Users/Shared/munki_repo',
 'MUNKI_REPO_SUBDIR': 'apps/Adobe',
 'NAME': 'AdobeReader',
 'OS_VERSION': 'Mac OS 12.0',
 'PARENT_RECIPES': ['/Users/n8felton/src/recipes/AdobeReader/AdobeReader.download.recipe'],
 'RECIPE_CACHE_DIR': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader',
 'RECIPE_DIR': '/Users/n8felton/src/recipes/AdobeReader',
 'RECIPE_OVERRIDE_DIRS': ['~/Library/AutoPkg/RecipeOverrides'],
 'RECIPE_PATH': '/Users/n8felton/src/recipes/AdobeReader/AdobeReader.munki.recipe',
 'RECIPE_SEARCH_DIRS': ['.',
                        '~/Library/AutoPkg/Recipes',
                        '/Library/AutoPkg/Recipes',
                        'AdobeReader'],
 'pkginfo': {'blocking_applications': ['Adobe Acrobat Pro.app',
                                       'Adobe Acrobat Reader DC.app',
                                       'Adobe Acrobat Reader.app',
                                       'Adobe Acrobat.app',
                                       'Adobe Reader.app'],
             'catalogs': ['testing'],
             'description': 'Adobe® Reader® is the global standard for '
                            'reliably viewing, printing, and commenting on PDF '
                            "documents. It's the only PDF file viewer that can "
                            'open and interact with all types of PDF content, '
                            'including forms and multimedia.',
             'display_name': 'Adobe Reader',
             'minimum_os_version': '10.15.0',
             'name': 'AdobeReader',
             'preinstall_script': '#!/bin/sh\n'
                                  'if [ -e "/Applications/Adobe Acrobat '
                                  'Reader.app" ]; then\n'
                                  '\trm -r "/Applications/Adobe Acrobat '
                                  'Reader.app"\n'
                                  'fi\n'
                                  'if [ -e "/Applications/Adobe Acrobat Reader '
                                  'DC.app" ]; then\n'
                                  '\trm -r "/Applications/Adobe Acrobat Reader '
                                  'DC.app"\n'
                                  'fi\n'
                                  'exit 0\n',
             'unattended_install': True},
 'verbose': 4}
AdobeReaderURLProvider
{'Input': {'os_version': 'Mac OS 12.0'}}
AdobeReaderURLProvider: RDC_PRODUCTS_URL: https://rdc.adobe.io/reader/products?os=Mac%20OS%2012.0&api_key=dc-get-adobereader-cdn
AdobeReaderURLProvider: Curl command: ['/usr/bin/curl', '--compressed', '--location', 'https://rdc.adobe.io/reader/products?os=Mac%20OS%2012.0&api_key=dc-get-adobereader-cdn']
AdobeReaderURLProvider: {'products': {'reader': [{'displayName': 'Reader DC 2022.003.20258 for Mac', 'fileSize': 342.1, 'version': '22.003.20258'}], 'dcPro': []}, 'addons': []}
AdobeReaderURLProvider: [displayName] : Reader DC 2022.003.20258 for Mac
AdobeReaderURLProvider: [version]     : 22.003.20258
AdobeReaderURLProvider: RDC_DOWNLOAD_URL: https://rdc.adobe.io/reader/downloadUrl?name=Reader%20DC%202022.003.20258%20for%20Mac&os=Mac%20OS%2012.0&api_key=dc-get-adobereader-cdn
AdobeReaderURLProvider: Curl command: ['/usr/bin/curl', '--compressed', '--location', 'https://rdc.adobe.io/reader/downloadUrl?name=Reader%20DC%202022.003.20258%20for%20Mac&os=Mac%20OS%2012.0&api_key=dc-get-adobereader-cdn']
AdobeReaderURLProvider: {'downloadURL': 'https://ardownload2.adobe.com/pub/adobe/reader/mac/AcrobatDC/2200320258/AcroRdrDC_2200320258_MUI.dmg', 'installer': 'standalone', 'saveName': 'AcroRdrDC_2200320258_MUI.dmg'}
AdobeReaderURLProvider: [download_url]: https://ardownload2.adobe.com/pub/adobe/reader/mac/AcrobatDC/2200320258/AcroRdrDC_2200320258_MUI.dmg
{'Output': {'filename': 'AcroRdrDC_2200320258_MUI.dmg',
            'url': 'https://ardownload2.adobe.com/pub/adobe/reader/mac/AcrobatDC/2200320258/AcroRdrDC_2200320258_MUI.dmg',
            'version': '22.003.20258'}}
URLDownloader
{'Input': {'filename': 'AdobeReader.dmg',
           'url': 'https://ardownload2.adobe.com/pub/adobe/reader/mac/AcrobatDC/2200320258/AcroRdrDC_2200320258_MUI.dmg'}}
URLDownloader: No value supplied for prefetch_filename, setting default value of: False
URLDownloader: No value supplied for CHECK_FILESIZE_ONLY, setting default value of: False
URLDownloader: Curl command: ['/usr/bin/curl', '--silent', '--show-error', '--no-buffer', '--dump-header', '-', '--speed-time', '30', '--location', '--url', 'https://ardownload2.adobe.com/pub/adobe/reader/mac/AcrobatDC/2200320258/AcroRdrDC_2200320258_MUI.dmg', '--fail', '--output', '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/downloads/tmp6uvxvqq9']
URLDownloader: Storing new Last-Modified header: Mon, 10 Oct 2022 06:15:51 GMT
URLDownloader: Storing new ETag header: "14646592-5eaa8176b80a7"
URLDownloader: Downloaded /Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/downloads/AdobeReader.dmg
{'Output': {'download_changed': True,
            'etag': '"14646592-5eaa8176b80a7"',
            'last_modified': 'Mon, 10 Oct 2022 06:15:51 GMT',
            'pathname': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/downloads/AdobeReader.dmg',
            'url_downloader_summary_result': {'data': {'download_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/downloads/AdobeReader.dmg'},
                                              'summary_text': 'The following '
                                                              'new items were '
                                                              'downloaded:'}}}
EndOfCheckPhase
{'Input': {}}
{'Output': {}}
CodeSignatureVerifier
{'Input': {'expected_authority_names': ['Developer ID Installer: Adobe Inc. '
                                        '(JQ525L2MZD)',
                                        'Developer ID Certification Authority',
                                        'Apple Root CA'],
           'input_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/downloads/AdobeReader.dmg/*.pkg'}}
CodeSignatureVerifier: Mounted disk image /Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/downloads/AdobeReader.dmg
CodeSignatureVerifier: Using path '/private/tmp/dmg.U8A6H5/AcroRdrDC_2200320258_MUI.pkg' matched from globbed '/private/tmp/dmg.U8A6H5/*.pkg'.
CodeSignatureVerifier: Verifying installer package signature...
CodeSignatureVerifier: Package "AcroRdrDC_2200320258_MUI.pkg":
CodeSignatureVerifier:    Status: signed by a developer certificate issued by Apple for distribution
CodeSignatureVerifier:    Notarization: trusted by the Apple notary service
CodeSignatureVerifier:    Signed with a trusted timestamp on: 2022-09-28 22:01:34 +0000
CodeSignatureVerifier:    Certificate Chain:
CodeSignatureVerifier:     1. Developer ID Installer: Adobe Inc. (JQ525L2MZD)
CodeSignatureVerifier:        Expires: 2027-02-01 22:12:15 +0000
CodeSignatureVerifier:        SHA256 Fingerprint:
CodeSignatureVerifier:            97 9A 81 BE 07 15 BA 87 99 0E AD DC AE 97 A7 98 08 19 8A 1E DE 0F 
CodeSignatureVerifier:            4A 42 0D CE 38 96 80 A1 CE BF
CodeSignatureVerifier:        ------------------------------------------------------------------------
CodeSignatureVerifier:     2. Developer ID Certification Authority
CodeSignatureVerifier:        Expires: 2027-02-01 22:12:15 +0000
CodeSignatureVerifier:        SHA256 Fingerprint:
CodeSignatureVerifier:            7A FC 9D 01 A6 2F 03 A2 DE 96 37 93 6D 4A FE 68 09 0D 2D E1 8D 03 
CodeSignatureVerifier:            F2 9C 88 CF B0 B1 BA 63 58 7F
CodeSignatureVerifier:        ------------------------------------------------------------------------
CodeSignatureVerifier:     3. Apple Root CA
CodeSignatureVerifier:        Expires: 2035-02-09 21:40:36 +0000
CodeSignatureVerifier:        SHA256 Fingerprint:
CodeSignatureVerifier:            B0 B1 73 0E CB C7 FF 45 05 14 2C 49 F1 29 5E 6E DA 6B CA ED 7E 2C 
CodeSignatureVerifier:            68 C5 BE 91 B5 A1 10 01 F0 24
CodeSignatureVerifier: 
CodeSignatureVerifier: Signature is valid
CodeSignatureVerifier: Authority name chain is valid
{'Output': {}}
FlatPkgUnpacker
{'Input': {'destination_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/pkg_unpack',
           'flat_pkg_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/downloads/AdobeReader.dmg/*.pkg',
           'purge_destination': True}}
FlatPkgUnpacker: Mounted disk image /Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/downloads/AdobeReader.dmg
FlatPkgUnpacker: Unpacked /private/tmp/dmg.Y159Xs/AcroRdrDC_2200320258_MUI.pkg to /Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/pkg_unpack
{'Output': {}}
PkgRootCreator
{'Input': {'pkgdirs': {},
           'pkgroot': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/application_payload/Applications'}}
PkgRootCreator: Created /Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/application_payload/Applications
{'Output': {}}
PkgPayloadUnpacker
{'Input': {'destination_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/application_payload/Applications',
           'pkg_payload_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/pkg_unpack/application.pkg/Payload',
           'purge_destination': True}}
PkgPayloadUnpacker: Unpacked /Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/pkg_unpack/application.pkg/Payload to /Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/application_payload/Applications
{'Output': {}}
FileFinder
{'Input': {'pattern': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/application_payload/Applications/*.app'}}
FileFinder: No value supplied for find_method, setting default value of: glob
FileFinder: Found file match: '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/application_payload/Applications/Adobe Acrobat Reader.app' from globbed '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/application_payload/Applications/*.app'
FileFinder: Basename match: 'Adobe Acrobat Reader.app'
{'Output': {'found_basename': 'Adobe Acrobat Reader.app',
            'found_filename': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/application_payload/Applications/Adobe '
                              'Acrobat Reader.app'}}
MunkiInstallsItemsCreator
{'Input': {'faux_root': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/application_payload',
           'installs_item_paths': ['/Applications/Adobe Acrobat Reader.app']}}
MunkiInstallsItemsCreator: Created installs item for /Applications/Adobe Acrobat Reader.app
{'Output': {'additional_pkginfo': {'installs': [{'CFBundleIdentifier': 'com.adobe.Reader',
                                                 'CFBundleName': 'Acrobat '
                                                                 'Reader',
                                                 'CFBundleShortVersionString': '22.003.20258',
                                                 'CFBundleVersion': '22.003.20258',
                                                 'minosversion': '10.13',
                                                 'path': '/Applications/Adobe '
                                                         'Acrobat Reader.app',
                                                 'type': 'application',
                                                 'version_comparison_key': 'CFBundleShortVersionString'}]}}}
MunkiPkginfoMerger
{'Input': {'additional_pkginfo': {'installs': [{'CFBundleIdentifier': 'com.adobe.Reader',
                                                'CFBundleName': 'Acrobat '
                                                                'Reader',
                                                'CFBundleShortVersionString': '22.003.20258',
                                                'CFBundleVersion': '22.003.20258',
                                                'minosversion': '10.13',
                                                'path': '/Applications/Adobe '
                                                        'Acrobat Reader.app',
                                                'type': 'application',
                                                'version_comparison_key': 'CFBundleShortVersionString'}]},
           'pkginfo': {'blocking_applications': ['Adobe Acrobat Pro.app',
                                                 'Adobe Acrobat Reader DC.app',
                                                 'Adobe Acrobat Reader.app',
                                                 'Adobe Acrobat.app',
                                                 'Adobe Reader.app'],
                       'catalogs': ['testing'],
                       'description': 'Adobe® Reader® is the global standard '
                                      'for reliably viewing, printing, and '
                                      "commenting on PDF documents. It's the "
                                      'only PDF file viewer that can open and '
                                      'interact with all types of PDF content, '
                                      'including forms and multimedia.',
                       'display_name': 'Adobe Reader',
                       'minimum_os_version': '10.15.0',
                       'name': 'AdobeReader',
                       'preinstall_script': '#!/bin/sh\n'
                                            'if [ -e "/Applications/Adobe '
                                            'Acrobat Reader.app" ]; then\n'
                                            '\trm -r "/Applications/Adobe '
                                            'Acrobat Reader.app"\n'
                                            'fi\n'
                                            'if [ -e "/Applications/Adobe '
                                            'Acrobat Reader DC.app" ]; then\n'
                                            '\trm -r "/Applications/Adobe '
                                            'Acrobat Reader DC.app"\n'
                                            'fi\n'
                                            'exit 0\n',
                       'unattended_install': True}}}
MunkiPkginfoMerger: Merged {'installs': [{'CFBundleIdentifier': 'com.adobe.Reader', 'CFBundleName': 'Acrobat Reader', 'CFBundleShortVersionString': '22.003.20258', 'CFBundleVersion': '22.003.20258', 'minosversion': '10.13', 'path': '/Applications/Adobe Acrobat Reader.app', 'type': 'application', 'version_comparison_key': 'CFBundleShortVersionString'}]} into pkginfo
{'Output': {'pkginfo': {'blocking_applications': ['Adobe Acrobat Pro.app',
                                                  'Adobe Acrobat Reader DC.app',
                                                  'Adobe Acrobat Reader.app',
                                                  'Adobe Acrobat.app',
                                                  'Adobe Reader.app'],
                        'catalogs': ['testing'],
                        'description': 'Adobe® Reader® is the global standard '
                                       'for reliably viewing, printing, and '
                                       "commenting on PDF documents. It's the "
                                       'only PDF file viewer that can open and '
                                       'interact with all types of PDF '
                                       'content, including forms and '
                                       'multimedia.',
                        'display_name': 'Adobe Reader',
                        'installs': [{'CFBundleIdentifier': 'com.adobe.Reader',
                                      'CFBundleName': 'Acrobat Reader',
                                      'CFBundleShortVersionString': '22.003.20258',
                                      'CFBundleVersion': '22.003.20258',
                                      'minosversion': '10.13',
                                      'path': '/Applications/Adobe Acrobat '
                                              'Reader.app',
                                      'type': 'application',
                                      'version_comparison_key': 'CFBundleShortVersionString'}],
                        'minimum_os_version': '10.15.0',
                        'name': 'AdobeReader',
                        'preinstall_script': '#!/bin/sh\n'
                                             'if [ -e "/Applications/Adobe '
                                             'Acrobat Reader.app" ]; then\n'
                                             '\trm -r "/Applications/Adobe '
                                             'Acrobat Reader.app"\n'
                                             'fi\n'
                                             'if [ -e "/Applications/Adobe '
                                             'Acrobat Reader DC.app" ]; then\n'
                                             '\trm -r "/Applications/Adobe '
                                             'Acrobat Reader DC.app"\n'
                                             'fi\n'
                                             'exit 0\n',
                        'unattended_install': True}}}
MunkiImporter
{'Input': {'MUNKI_REPO': '/Users/Shared/munki_repo',
           'pkg_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/downloads/AdobeReader.dmg',
           'pkginfo': {'blocking_applications': ['Adobe Acrobat Pro.app',
                                                 'Adobe Acrobat Reader DC.app',
                                                 'Adobe Acrobat Reader.app',
                                                 'Adobe Acrobat.app',
                                                 'Adobe Reader.app'],
                       'catalogs': ['testing'],
                       'description': 'Adobe® Reader® is the global standard '
                                      'for reliably viewing, printing, and '
                                      "commenting on PDF documents. It's the "
                                      'only PDF file viewer that can open and '
                                      'interact with all types of PDF content, '
                                      'including forms and multimedia.',
                       'display_name': 'Adobe Reader',
                       'installs': [{'CFBundleIdentifier': 'com.adobe.Reader',
                                     'CFBundleName': 'Acrobat Reader',
                                     'CFBundleShortVersionString': '22.003.20258',
                                     'CFBundleVersion': '22.003.20258',
                                     'minosversion': '10.13',
                                     'path': '/Applications/Adobe Acrobat '
                                             'Reader.app',
                                     'type': 'application',
                                     'version_comparison_key': 'CFBundleShortVersionString'}],
                       'minimum_os_version': '10.15.0',
                       'name': 'AdobeReader',
                       'preinstall_script': '#!/bin/sh\n'
                                            'if [ -e "/Applications/Adobe '
                                            'Acrobat Reader.app" ]; then\n'
                                            '\trm -r "/Applications/Adobe '
                                            'Acrobat Reader.app"\n'
                                            'fi\n'
                                            'if [ -e "/Applications/Adobe '
                                            'Acrobat Reader DC.app" ]; then\n'
                                            '\trm -r "/Applications/Adobe '
                                            'Acrobat Reader DC.app"\n'
                                            'fi\n'
                                            'exit 0\n',
                       'unattended_install': True},
           'repo_subdirectory': 'apps/Adobe'}}
MunkiImporter: No value supplied for MUNKI_REPO_PLUGIN, setting default value of: FileRepo
MunkiImporter: No value supplied for MUNKILIB_DIR, setting default value of: /usr/local/munki
MunkiImporter: No value supplied for force_munki_repo_lib, setting default value of: False
MunkiImporter: Using repo lib: AutoPkgLib
MunkiImporter:         plugin: FileRepo
MunkiImporter:           repo: /Users/Shared/munki_repo
MunkiImporter: Copied pkginfo to: /Users/Shared/munki_repo/pkgsinfo/apps/Adobe/AdobeReader-22.003.20258.plist
MunkiImporter:            pkg to: /Users/Shared/munki_repo/pkgs/apps/Adobe/AdobeReader-22.003.20258.dmg
{'Output': {'munki_importer_summary_result': {'data': {'catalogs': 'testing',
                                                       'icon_repo_path': '',
                                                       'name': 'AdobeReader',
                                                       'pkg_repo_path': 'apps/Adobe/AdobeReader-22.003.20258.dmg',
                                                       'pkginfo_path': 'apps/Adobe/AdobeReader-22.003.20258.plist',
                                                       'version': '22.003.20258'},
                                              'report_fields': ['name',
                                                                'version',
                                                                'catalogs',
                                                                'pkginfo_path',
                                                                'pkg_repo_path',
                                                                'icon_repo_path'],
                                              'summary_text': 'The following '
                                                              'new items were '
                                                              'imported into '
                                                              'Munki:'},
            'munki_info': {'_metadata': {'created_by': 'n8felton',
                                         'creation_date': datetime.datetime(2022, 10, 14, 20, 9, 29),
                                         'munki_version': '5.7.3.4443',
                                         'os_version': '12.6'},
                           'autoremove': False,
                           'blocking_applications': ['Adobe Acrobat Pro.app',
                                                     'Adobe Acrobat Reader '
                                                     'DC.app',
                                                     'Adobe Acrobat Reader.app',
                                                     'Adobe Acrobat.app',
                                                     'Adobe Reader.app'],
                           'catalogs': ['testing'],
                           'description': 'Adobe® Reader® is the global '
                                          'standard for reliably viewing, '
                                          'printing, and commenting on PDF '
                                          "documents. It's the only PDF file "
                                          'viewer that can open and interact '
                                          'with all types of PDF content, '
                                          'including forms and multimedia.',
                           'display_name': 'Adobe Reader',
                           'installed_size': 392981,
                           'installer_item_hash': 'f23d3f6a6c10e2df540c2345d8b1c90e28b9ef6de7b9d63739ed9b715b2b7ecc',
                           'installer_item_location': 'apps/Adobe/AdobeReader-22.003.20258.dmg',
                           'installer_item_size': 334105,
                           'installs': [{'CFBundleIdentifier': 'com.adobe.Reader',
                                         'CFBundleName': 'Acrobat Reader',
                                         'CFBundleShortVersionString': '22.003.20258',
                                         'CFBundleVersion': '22.003.20258',
                                         'minosversion': '10.13',
                                         'path': '/Applications/Adobe Acrobat '
                                                 'Reader.app',
                                         'type': 'application',
                                         'version_comparison_key': 'CFBundleShortVersionString'}],
                           'minimum_os_version': '10.15.0',
                           'name': 'AdobeReader',
                           'preinstall_script': '#!/bin/sh\n'
                                                'if [ -e "/Applications/Adobe '
                                                'Acrobat Reader.app" ]; then\n'
                                                '\trm -r "/Applications/Adobe '
                                                'Acrobat Reader.app"\n'
                                                'fi\n'
                                                'if [ -e "/Applications/Adobe '
                                                'Acrobat Reader DC.app" ]; '
                                                'then\n'
                                                '\trm -r "/Applications/Adobe '
                                                'Acrobat Reader DC.app"\n'
                                                'fi\n'
                                                'exit 0\n',
                           'receipts': [{'installed_size': 48923,
                                         'packageid': 'com.adobe.RdrServicesUpdater',
                                         'version': '0'},
                                        {'installed_size': 243839,
                                         'packageid': 'com.adobe.acrobat.DC.reader.app.pkg.MUI',
                                         'version': '22.003.20258'},
                                        {'installed_size': 98872,
                                         'packageid': 'com.adobe.acrobat.DC.reader.appsupport.pkg.MUI',
                                         'version': '22.003.20258'},
                                        {'installed_size': 1347,
                                         'packageid': 'com.adobe.armdc.app.pkg',
                                         'version': '1.0.0'}],
                           'unattended_install': True,
                           'uninstall_method': 'removepackages',
                           'uninstallable': True,
                           'version': '22.003.20258'},
            'munki_repo_changed': True,
            'pkg_repo_path': '/Users/Shared/munki_repo/pkgs/apps/Adobe/AdobeReader-22.003.20258.dmg',
            'pkginfo_repo_path': '/Users/Shared/munki_repo/pkgsinfo/apps/Adobe/AdobeReader-22.003.20258.plist'}}
{'AUTOPKG_VERSION': '2.7',
 'CHECK_FILESIZE_ONLY': False,
 'MUNKILIB_DIR': '/usr/local/munki',
 'MUNKI_REPO': '/Users/Shared/munki_repo',
 'MUNKI_REPO_PLUGIN': 'FileRepo',
 'MUNKI_REPO_SUBDIR': 'apps/Adobe',
 'NAME': 'AdobeReader',
 'OS_VERSION': 'Mac OS 12.0',
 'PARENT_RECIPES': ['/Users/n8felton/src/recipes/AdobeReader/AdobeReader.download.recipe'],
 'RECIPE_CACHE_DIR': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader',
 'RECIPE_DIR': '/Users/n8felton/src/recipes/AdobeReader',
 'RECIPE_OVERRIDE_DIRS': ['~/Library/AutoPkg/RecipeOverrides'],
 'RECIPE_PATH': '/Users/n8felton/src/recipes/AdobeReader/AdobeReader.munki.recipe',
 'RECIPE_SEARCH_DIRS': ['.',
                        '~/Library/AutoPkg/Recipes',
                        '/Library/AutoPkg/Recipes',
                        'AdobeReader'],
 'additional_pkginfo': {'installs': [{'CFBundleIdentifier': 'com.adobe.Reader',
                                      'CFBundleName': 'Acrobat Reader',
                                      'CFBundleShortVersionString': '22.003.20258',
                                      'CFBundleVersion': '22.003.20258',
                                      'minosversion': '10.13',
                                      'path': '/Applications/Adobe Acrobat '
                                              'Reader.app',
                                      'type': 'application',
                                      'version_comparison_key': 'CFBundleShortVersionString'}]},
 'destination_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/application_payload/Applications',
 'download_changed': True,
 'etag': '"14646592-5eaa8176b80a7"',
 'expected_authority_names': ['Developer ID Installer: Adobe Inc. (JQ525L2MZD)',
                              'Developer ID Certification Authority',
                              'Apple Root CA'],
 'faux_root': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/application_payload',
 'filename': 'AdobeReader.dmg',
 'find_method': 'glob',
 'flat_pkg_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/downloads/AdobeReader.dmg/*.pkg',
 'force_munki_repo_lib': False,
 'found_basename': 'Adobe Acrobat Reader.app',
 'found_filename': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/application_payload/Applications/Adobe '
                   'Acrobat Reader.app',
 'icon_repo_path': '',
 'input_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/downloads/AdobeReader.dmg/*.pkg',
 'installs_item_paths': ['/Applications/Adobe Acrobat Reader.app'],
 'last_modified': 'Mon, 10 Oct 2022 06:15:51 GMT',
 'munki_importer_summary_result': {'data': {'catalogs': 'testing',
                                            'icon_repo_path': '',
                                            'name': 'AdobeReader',
                                            'pkg_repo_path': 'apps/Adobe/AdobeReader-22.003.20258.dmg',
                                            'pkginfo_path': 'apps/Adobe/AdobeReader-22.003.20258.plist',
                                            'version': '22.003.20258'},
                                   'report_fields': ['name',
                                                     'version',
                                                     'catalogs',
                                                     'pkginfo_path',
                                                     'pkg_repo_path',
                                                     'icon_repo_path'],
                                   'summary_text': 'The following new items '
                                                   'were imported into Munki:'},
 'munki_info': {'_metadata': {'created_by': 'n8felton',
                              'creation_date': datetime.datetime(2022, 10, 14, 20, 9, 29),
                              'munki_version': '5.7.3.4443',
                              'os_version': '12.6'},
                'autoremove': False,
                'blocking_applications': ['Adobe Acrobat Pro.app',
                                          'Adobe Acrobat Reader DC.app',
                                          'Adobe Acrobat Reader.app',
                                          'Adobe Acrobat.app',
                                          'Adobe Reader.app'],
                'catalogs': ['testing'],
                'description': 'Adobe® Reader® is the global standard for '
                               'reliably viewing, printing, and commenting on '
                               "PDF documents. It's the only PDF file viewer "
                               'that can open and interact with all types of '
                               'PDF content, including forms and multimedia.',
                'display_name': 'Adobe Reader',
                'installed_size': 392981,
                'installer_item_hash': 'f23d3f6a6c10e2df540c2345d8b1c90e28b9ef6de7b9d63739ed9b715b2b7ecc',
                'installer_item_location': 'apps/Adobe/AdobeReader-22.003.20258.dmg',
                'installer_item_size': 334105,
                'installs': [{'CFBundleIdentifier': 'com.adobe.Reader',
                              'CFBundleName': 'Acrobat Reader',
                              'CFBundleShortVersionString': '22.003.20258',
                              'CFBundleVersion': '22.003.20258',
                              'minosversion': '10.13',
                              'path': '/Applications/Adobe Acrobat Reader.app',
                              'type': 'application',
                              'version_comparison_key': 'CFBundleShortVersionString'}],
                'minimum_os_version': '10.15.0',
                'name': 'AdobeReader',
                'preinstall_script': '#!/bin/sh\n'
                                     'if [ -e "/Applications/Adobe Acrobat '
                                     'Reader.app" ]; then\n'
                                     '\trm -r "/Applications/Adobe Acrobat '
                                     'Reader.app"\n'
                                     'fi\n'
                                     'if [ -e "/Applications/Adobe Acrobat '
                                     'Reader DC.app" ]; then\n'
                                     '\trm -r "/Applications/Adobe Acrobat '
                                     'Reader DC.app"\n'
                                     'fi\n'
                                     'exit 0\n',
                'receipts': [{'installed_size': 48923,
                              'packageid': 'com.adobe.RdrServicesUpdater',
                              'version': '0'},
                             {'installed_size': 243839,
                              'packageid': 'com.adobe.acrobat.DC.reader.app.pkg.MUI',
                              'version': '22.003.20258'},
                             {'installed_size': 98872,
                              'packageid': 'com.adobe.acrobat.DC.reader.appsupport.pkg.MUI',
                              'version': '22.003.20258'},
                             {'installed_size': 1347,
                              'packageid': 'com.adobe.armdc.app.pkg',
                              'version': '1.0.0'}],
                'unattended_install': True,
                'uninstall_method': 'removepackages',
                'uninstallable': True,
                'version': '22.003.20258'},
 'munki_repo_changed': True,
 'os_version': 'Mac OS 12.0',
 'pathname': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/downloads/AdobeReader.dmg',
 'pattern': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/application_payload/Applications/*.app',
 'pkg_path': '/Users/Shared/munki_repo/pkgs/apps/Adobe/AdobeReader-22.003.20258.dmg',
 'pkg_payload_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/pkg_unpack/application.pkg/Payload',
 'pkg_repo_path': '/Users/Shared/munki_repo/pkgs/apps/Adobe/AdobeReader-22.003.20258.dmg',
 'pkgdirs': {},
 'pkginfo': {'blocking_applications': ['Adobe Acrobat Pro.app',
                                       'Adobe Acrobat Reader DC.app',
                                       'Adobe Acrobat Reader.app',
                                       'Adobe Acrobat.app',
                                       'Adobe Reader.app'],
             'catalogs': ['testing'],
             'description': 'Adobe® Reader® is the global standard for '
                            'reliably viewing, printing, and commenting on PDF '
                            "documents. It's the only PDF file viewer that can "
                            'open and interact with all types of PDF content, '
                            'including forms and multimedia.',
             'display_name': 'Adobe Reader',
             'installs': [{'CFBundleIdentifier': 'com.adobe.Reader',
                           'CFBundleName': 'Acrobat Reader',
                           'CFBundleShortVersionString': '22.003.20258',
                           'CFBundleVersion': '22.003.20258',
                           'minosversion': '10.13',
                           'path': '/Applications/Adobe Acrobat Reader.app',
                           'type': 'application',
                           'version_comparison_key': 'CFBundleShortVersionString'}],
             'minimum_os_version': '10.15.0',
             'name': 'AdobeReader',
             'preinstall_script': '#!/bin/sh\n'
                                  'if [ -e "/Applications/Adobe Acrobat '
                                  'Reader.app" ]; then\n'
                                  '\trm -r "/Applications/Adobe Acrobat '
                                  'Reader.app"\n'
                                  'fi\n'
                                  'if [ -e "/Applications/Adobe Acrobat Reader '
                                  'DC.app" ]; then\n'
                                  '\trm -r "/Applications/Adobe Acrobat Reader '
                                  'DC.app"\n'
                                  'fi\n'
                                  'exit 0\n',
             'unattended_install': True},
 'pkginfo_repo_path': '/Users/Shared/munki_repo/pkgsinfo/apps/Adobe/AdobeReader-22.003.20258.plist',
 'pkgroot': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/application_payload/Applications',
 'prefetch_filename': False,
 'purge_destination': True,
 'repo_subdirectory': 'apps/Adobe',
 'url': 'https://ardownload2.adobe.com/pub/adobe/reader/mac/AcrobatDC/2200320258/AcroRdrDC_2200320258_MUI.dmg',
 'url_downloader_summary_result': {'data': {'download_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/downloads/AdobeReader.dmg'},
                                   'summary_text': 'The following new items '
                                                   'were downloaded:'},
 'verbose': 4,
 'version': '22.003.20258'}
Receipt written to /Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/receipts/AdobeReader.munki-receipt-20221014-130929.plist

The following new items were downloaded:
    Download Path                                                                                         
    -------------                                                                                         
    /Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.munki.AdobeReader/downloads/AdobeReader.dmg  

The following new items were imported into Munki:
    Name         Version       Catalogs  Pkginfo Path                               Pkg Repo Path                            Icon Repo Path  
    ----         -------       --------  ------------                               -------------                            --------------  
    AdobeReader  22.003.20258  testing   apps/Adobe/AdobeReader-22.003.20258.plist  apps/Adobe/AdobeReader-22.003.20258.dmg                  
```

n8felton commented:
```
% autopkg run -vvvv AdobeReader/AdobeReader.pkg.recipe                               
Processing AdobeReader/AdobeReader.pkg.recipe...
WARNING: AdobeReader/AdobeReader.pkg.recipe is missing trust info and FAIL_RECIPES_WITHOUT_TRUST_INFO is not set. Proceeding...
{'AUTOPKG_VERSION': '2.7',
 'MUNKI_REPO': '/Users/Shared/munki_repo',
 'NAME': 'AdobeReader',
 'OS_VERSION': 'Mac OS 12.0',
 'PARENT_RECIPES': ['/Users/n8felton/src/recipes/AdobeReader/AdobeReader.download.recipe'],
 'RECIPE_CACHE_DIR': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader',
 'RECIPE_DIR': '/Users/n8felton/src/recipes/AdobeReader',
 'RECIPE_OVERRIDE_DIRS': ['~/Library/AutoPkg/RecipeOverrides'],
 'RECIPE_PATH': '/Users/n8felton/src/recipes/AdobeReader/AdobeReader.pkg.recipe',
 'RECIPE_SEARCH_DIRS': ['.',
                        '~/Library/AutoPkg/Recipes',
                        '/Library/AutoPkg/Recipes',
                        'AdobeReader'],
 'verbose': 4}
AdobeReaderURLProvider
{'Input': {'os_version': 'Mac OS 12.0'}}
AdobeReaderURLProvider: RDC_PRODUCTS_URL: https://rdc.adobe.io/reader/products?os=Mac%20OS%2012.0&api_key=dc-get-adobereader-cdn
AdobeReaderURLProvider: Curl command: ['/usr/bin/curl', '--compressed', '--location', 'https://rdc.adobe.io/reader/products?os=Mac%20OS%2012.0&api_key=dc-get-adobereader-cdn']
AdobeReaderURLProvider: {'products': {'reader': [{'displayName': 'Reader DC 2022.003.20258 for Mac', 'fileSize': 342.1, 'version': '22.003.20258'}], 'dcPro': []}, 'addons': []}
AdobeReaderURLProvider: [displayName] : Reader DC 2022.003.20258 for Mac
AdobeReaderURLProvider: [version]     : 22.003.20258
AdobeReaderURLProvider: RDC_DOWNLOAD_URL: https://rdc.adobe.io/reader/downloadUrl?name=Reader%20DC%202022.003.20258%20for%20Mac&os=Mac%20OS%2012.0&api_key=dc-get-adobereader-cdn
AdobeReaderURLProvider: Curl command: ['/usr/bin/curl', '--compressed', '--location', 'https://rdc.adobe.io/reader/downloadUrl?name=Reader%20DC%202022.003.20258%20for%20Mac&os=Mac%20OS%2012.0&api_key=dc-get-adobereader-cdn']
AdobeReaderURLProvider: {'downloadURL': 'https://ardownload2.adobe.com/pub/adobe/reader/mac/AcrobatDC/2200320258/AcroRdrDC_2200320258_MUI.dmg', 'installer': 'standalone', 'saveName': 'AcroRdrDC_2200320258_MUI.dmg'}
AdobeReaderURLProvider: [download_url]: https://ardownload2.adobe.com/pub/adobe/reader/mac/AcrobatDC/2200320258/AcroRdrDC_2200320258_MUI.dmg
{'Output': {'filename': 'AcroRdrDC_2200320258_MUI.dmg',
            'url': 'https://ardownload2.adobe.com/pub/adobe/reader/mac/AcrobatDC/2200320258/AcroRdrDC_2200320258_MUI.dmg',
            'version': '22.003.20258'}}
URLDownloader
{'Input': {'filename': 'AdobeReader.dmg',
           'url': 'https://ardownload2.adobe.com/pub/adobe/reader/mac/AcrobatDC/2200320258/AcroRdrDC_2200320258_MUI.dmg'}}
URLDownloader: No value supplied for prefetch_filename, setting default value of: False
URLDownloader: No value supplied for CHECK_FILESIZE_ONLY, setting default value of: False
URLDownloader: Curl command: ['/usr/bin/curl', '--silent', '--show-error', '--no-buffer', '--dump-header', '-', '--speed-time', '30', '--location', '--url', 'https://ardownload2.adobe.com/pub/adobe/reader/mac/AcrobatDC/2200320258/AcroRdrDC_2200320258_MUI.dmg', '--fail', '--output', '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/downloads/tmp5477c7qa']
URLDownloader: Storing new Last-Modified header: Mon, 10 Oct 2022 06:15:51 GMT
URLDownloader: Storing new ETag header: "14646592-5eaa8176b80a7"
URLDownloader: Downloaded /Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/downloads/AdobeReader.dmg
{'Output': {'download_changed': True,
            'etag': '"14646592-5eaa8176b80a7"',
            'last_modified': 'Mon, 10 Oct 2022 06:15:51 GMT',
            'pathname': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/downloads/AdobeReader.dmg',
            'url_downloader_summary_result': {'data': {'download_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/downloads/AdobeReader.dmg'},
                                              'summary_text': 'The following '
                                                              'new items were '
                                                              'downloaded:'}}}
EndOfCheckPhase
{'Input': {}}
{'Output': {}}
CodeSignatureVerifier
{'Input': {'expected_authority_names': ['Developer ID Installer: Adobe Inc. '
                                        '(JQ525L2MZD)',
                                        'Developer ID Certification Authority',
                                        'Apple Root CA'],
           'input_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/downloads/AdobeReader.dmg/*.pkg'}}
CodeSignatureVerifier: Mounted disk image /Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/downloads/AdobeReader.dmg
CodeSignatureVerifier: Using path '/private/tmp/dmg.T72HkE/AcroRdrDC_2200320258_MUI.pkg' matched from globbed '/private/tmp/dmg.T72HkE/*.pkg'.
CodeSignatureVerifier: Verifying installer package signature...
CodeSignatureVerifier: Package "AcroRdrDC_2200320258_MUI.pkg":
CodeSignatureVerifier:    Status: signed by a developer certificate issued by Apple for distribution
CodeSignatureVerifier:    Notarization: trusted by the Apple notary service
CodeSignatureVerifier:    Signed with a trusted timestamp on: 2022-09-28 22:01:34 +0000
CodeSignatureVerifier:    Certificate Chain:
CodeSignatureVerifier:     1. Developer ID Installer: Adobe Inc. (JQ525L2MZD)
CodeSignatureVerifier:        Expires: 2027-02-01 22:12:15 +0000
CodeSignatureVerifier:        SHA256 Fingerprint:
CodeSignatureVerifier:            97 9A 81 BE 07 15 BA 87 99 0E AD DC AE 97 A7 98 08 19 8A 1E DE 0F 
CodeSignatureVerifier:            4A 42 0D CE 38 96 80 A1 CE BF
CodeSignatureVerifier:        ------------------------------------------------------------------------
CodeSignatureVerifier:     2. Developer ID Certification Authority
CodeSignatureVerifier:        Expires: 2027-02-01 22:12:15 +0000
CodeSignatureVerifier:        SHA256 Fingerprint:
CodeSignatureVerifier:            7A FC 9D 01 A6 2F 03 A2 DE 96 37 93 6D 4A FE 68 09 0D 2D E1 8D 03 
CodeSignatureVerifier:            F2 9C 88 CF B0 B1 BA 63 58 7F
CodeSignatureVerifier:        ------------------------------------------------------------------------
CodeSignatureVerifier:     3. Apple Root CA
CodeSignatureVerifier:        Expires: 2035-02-09 21:40:36 +0000
CodeSignatureVerifier:        SHA256 Fingerprint:
CodeSignatureVerifier:            B0 B1 73 0E CB C7 FF 45 05 14 2C 49 F1 29 5E 6E DA 6B CA ED 7E 2C 
CodeSignatureVerifier:            68 C5 BE 91 B5 A1 10 01 F0 24
CodeSignatureVerifier: 
CodeSignatureVerifier: Signature is valid
CodeSignatureVerifier: Authority name chain is valid
{'Output': {}}
AdobeReaderRepackager
{'Input': {'dmg_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/downloads/AdobeReader.dmg'}}
AdobeReaderRepackager: Mounted disk image /Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/downloads/AdobeReader.dmg
AdobeReaderRepackager: Replaced pkg preinstall script with our custom script at /Users/n8felton/src/recipes/AdobeReader/package_resources/scripts/readerdc_preinstall
{'Output': {'pkg_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/AcroRdrDC_2200320258_MUI.pkg'}}
FileFinder
{'Input': {'pattern': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/*/application.pkg'}}
FileFinder: No value supplied for find_method, setting default value of: glob
FileFinder: Found file match: '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/AcroRdrDC_2200320258_MUI/application.pkg' from globbed '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/*/application.pkg'
FileFinder: Basename match: 'application.pkg'
{'Output': {'found_basename': 'application.pkg',
            'found_filename': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/AcroRdrDC_2200320258_MUI/application.pkg'}}
PkgPayloadUnpacker
{'Input': {'destination_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader',
           'pkg_payload_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/AcroRdrDC_2200320258_MUI/application.pkg/Payload'}}
PkgPayloadUnpacker: Unpacked /Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/AcroRdrDC_2200320258_MUI/application.pkg/Payload to /Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader
{'Output': {}}
FileFinder
{'Input': {'find_method': 'glob',
           'pattern': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/*.app'}}
FileFinder: Found file match: '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/Adobe Acrobat Reader.app' from globbed '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/*.app'
FileFinder: Basename match: 'Adobe Acrobat Reader.app'
{'Output': {'found_basename': 'Adobe Acrobat Reader.app',
            'found_filename': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/Adobe '
                              'Acrobat Reader.app'}}
Versioner
{'Input': {'input_plist_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/Adobe '
                               'Acrobat Reader.app/Contents/Info.plist'}}
Versioner: No value supplied for plist_version_key, setting default value of: CFBundleShortVersionString
Versioner: No value supplied for skip_single_root_dir, setting default value of: False
Versioner: Found version 22.003.20258 in file /Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/Adobe Acrobat Reader.app/Contents/Info.plist
{'Output': {'version': '22.003.20258'}}
PathDeleter
{'Input': {'path_list': ['/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/Adobe '
                         'Acrobat Reader.app']}}
PathDeleter: Deleted /Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/Adobe Acrobat Reader.app
{'Output': {}}
{'AUTOPKG_VERSION': '2.7',
 'CHECK_FILESIZE_ONLY': False,
 'MUNKI_REPO': '/Users/Shared/munki_repo',
 'NAME': 'AdobeReader',
 'OS_VERSION': 'Mac OS 12.0',
 'PARENT_RECIPES': ['/Users/n8felton/src/recipes/AdobeReader/AdobeReader.download.recipe'],
 'RECIPE_CACHE_DIR': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader',
 'RECIPE_DIR': '/Users/n8felton/src/recipes/AdobeReader',
 'RECIPE_OVERRIDE_DIRS': ['~/Library/AutoPkg/RecipeOverrides'],
 'RECIPE_PATH': '/Users/n8felton/src/recipes/AdobeReader/AdobeReader.pkg.recipe',
 'RECIPE_SEARCH_DIRS': ['.',
                        '~/Library/AutoPkg/Recipes',
                        '/Library/AutoPkg/Recipes',
                        'AdobeReader'],
 'destination_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader',
 'dmg_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/downloads/AdobeReader.dmg',
 'download_changed': True,
 'etag': '"14646592-5eaa8176b80a7"',
 'expected_authority_names': ['Developer ID Installer: Adobe Inc. (JQ525L2MZD)',
                              'Developer ID Certification Authority',
                              'Apple Root CA'],
 'filename': 'AdobeReader.dmg',
 'find_method': 'glob',
 'found_basename': 'Adobe Acrobat Reader.app',
 'found_filename': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/Adobe '
                   'Acrobat Reader.app',
 'input_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/downloads/AdobeReader.dmg/*.pkg',
 'input_plist_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/Adobe '
                     'Acrobat Reader.app/Contents/Info.plist',
 'last_modified': 'Mon, 10 Oct 2022 06:15:51 GMT',
 'os_version': 'Mac OS 12.0',
 'path_list': ['/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/Adobe '
               'Acrobat Reader.app'],
 'pathname': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/downloads/AdobeReader.dmg',
 'pattern': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/*.app',
 'pkg_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/AcroRdrDC_2200320258_MUI.pkg',
 'pkg_payload_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/AcroRdrDC_2200320258_MUI/application.pkg/Payload',
 'plist_version_key': 'CFBundleShortVersionString',
 'prefetch_filename': False,
 'skip_single_root_dir': False,
 'url': 'https://ardownload2.adobe.com/pub/adobe/reader/mac/AcrobatDC/2200320258/AcroRdrDC_2200320258_MUI.dmg',
 'url_downloader_summary_result': {'data': {'download_path': '/Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/downloads/AdobeReader.dmg'},
                                   'summary_text': 'The following new items '
                                                   'were downloaded:'},
 'verbose': 4,
 'version': '22.003.20258'}
Receipt written to /Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/receipts/AdobeReader.pkg-receipt-20221014-131145.plist

The following new items were downloaded:
    Download Path                                                                                       
    -------------                                                                                       
    /Users/n8felton/Library/AutoPkg/Cache/com.github.autopkg.pkg.AdobeReader/downloads/AdobeReader.dmg  
```

n8felton commented:
AdobeReader.install.recipe is currently untested as I'm having issues with install recipes right now.

mpanighetti reviewed (approved):
Not sure who this is waiting on for review, but for what it's worth, I've successfully tested AdobeReader.download and AdobeReader.pkg.
